### PR TITLE
Changes in data retrieval

### PR DESCRIPTION
**Pull request by BirgitPfarrkirchner**
@forderud 
Suggested change: Acquire the three planes for the depiction separately with a modified bounding box instead of plane extraction from one data block.

### CODE REVIEW

**Comment by forderud** [comment]
Good idea @BirgitPfarrkirchner . I like what you're doing here.

The three scopes/loops that creates & fills a `WriteableBitmap` now appear very similar. Do you think you could also try to factor out the code into a common function, so that we reduce duplication?

**Comment by BirgitPfarrkirchner** [comment]
Thanks @forderud, that's a good idea. I will create a function for that.

File: TestViewer/MainWindow.xaml [hunk 1]
```diff
@@ -15,7 +15,7 @@
         <StackPanel Grid.Column="0">
             <!-- Menu pane -->
             <Label>Loader name</Label>
-            <TextBox x:Name="LoaderName">DummyLoader.Image3dFileLoader</TextBox>
+            <TextBox x:Name="LoaderName">GEHC_CARD_US.Image3dFileLoader</TextBox>
```
Comment (forderud) [inline TestViewer/MainWindow.xaml]: Can you please keep "DummyLoader.Image3dFileLoader" as default value, so that the application works out-of-the-box also for users without access to the GE CVUS loader.

You can convert the TextBox into a pull-down menu or similar, where "GEHC_CARD_US.Image3dFileLoader" is one of the options, if you want to ease access to the GE CVUS loader. Please do that in a separate PR.
